### ISSUE TITLE
[GraphQL] Request for Contributors do not returns them unless they are filtered

### ISSUE THREAD

**Issue by kjgarza** [datacite/lupo#630]
### Expected Behaviour

Request for contributors should return all contributors 
for example https://api.datacite.org/dois/10.48321/D1H59R

```
{
 dataManagementPlan(id: "10.48321/D1H59R") {
    contributors {
      id
      name
      affiliation{
        id
      }
    }  
  }
}
```

### Current Behaviour

it returns none but it kind of works if you add a filter. see example below:

```
{
 dataManagementPlan(id: "10.48321/D1H59R") {
    contributors(contributorType: "HostingInstitution") {
      id
      name
      affiliation{
        id
      }
    }  
  }
}

```

### Steps to Reproduce

1. step1
2. step2
3. step3
4. step4

**Comment by mfenner** [comment]
Implemented via #632.

**Comment by kjgarza** [comment]
@mfenner  This is still returning nothing when unfiltered.

![image](https://user-images.githubusercontent.com/1092861/93451903-08a6e080-f8d8-11ea-8757-c2c84579ad38.png)

I tried updating the DOI but still it did not work. I can see the test in the code works (#632) but it doesn't seem to be working in production.
